spec/index.html

```diff
@@ -1867,9 +1980,12 @@ <h3>Plain Literal Event</h3>
 
       <p class="note">RDF/XML plain literals are in RDF 1.2 treated as
         syntactic sugar for a literal with datatype
-        <code>xsd:string</code> (in case no language tag is present)
-        or as a literal with datatype <code>rdf:langString</code> (in
-        case a language tag is present). The mapping to N-Triples as
+        <code>xsd:string</code> (when no language tag is present),
+        as a literal with datatype <code>rdf:langString</code> (when
+        a language tag is present and a direction is not present),
+        or as a literal with datatype <code>rdf:dirLangString</code>
+        (when both a language tag and a direction are present).
```

Comment:
That does not tell us what to do if we have a direction but no language tag...

Reply:
The result would be illegal according to Concepts, but the triple generation wording in this spec is archaic and should be taken up separately.